MapboxAndroidDemo/build.gradle

```diff
@@ -2,6 +2,7 @@ apply plugin: 'com.android.application'
 apply plugin: 'com.google.firebase.firebase-perf'
 apply plugin: 'com.github.triplet.play'
 apply from: "$project.rootDir/gradle/script-git-version.gradle"
+apply from: "$project.rootDir/gradle/graph.gradle"
```

Comment:
In order to understand in a glance what the plugin does, what about calling it `dependencies-graph.gradle` instead?

Reply:
Renamed to `gradle-dependencies-graph.gradle`. Looks like the Maps SDK likes prefixing all scripts with `gradle-`.